test/built-ins/TypedArrays/from/mapfn-is-not-callable.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+id: sec-%typedarray%.from
+description: Throw a TypeError exception is mapfn is not callable
+info: >
+  22.2.2.1 %TypedArray%.from ( source [ , mapfn [ , thisArg ] ] )
+
+  ...
+  3. If mapfn was supplied and mapfn is not undefined, then
+    a. If IsCallable(mapfn) is false, throw a TypeError exception.
+  ...
+includes: [testTypedArray.js]
+features: [Symbol.iterator]
+---*/
+
+var arrayLike = {};
+Object.defineProperty(arrayLike, Symbol.iterator, {
```

Comment:
it was intentional. The next step with an abrupt completion would be the getting Symbol.iterator from the source arg.

```
6. Let arrayLike be ? IterableToArrayLike(source).
```

I'm asserting the TypeError happens and it comes before the Test262Error.